README.md

```diff
@@ -1 +1,125 @@
-# terraform-aws-iam-chamber-user
\ No newline at end of file
+# terraform-aws-iam-chamber-user [![Build Status](https://travis-ci.org/cloudposse/terraform-aws-iam-chamber-user.svg)](https://travis-ci.org/cloudposse/terraform-aws-iam-chamber-user)
+
+Terraform Module to provision a basic IAM [chamber](https://github.com/segmentio/chamber) user with access to SSM parameters and KMS key to decrypt secrets, suitable for CI/CD Systems
+(_e.g._ TravisCI, CircleCI, CodeFresh) or systems which are *external* to AWS that cannot leverage [AWS IAM Instance Profiles](http://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_use_switch-role-ec2_instance-profiles.html).
+
+We do not recommend creating IAM users this way for any other purpose.
+
+## Usage
+
+### Simple usage
+
+```hcl
+module "circleci" {
+  source     = "git::https://github.com/cloudposse/terraform-aws-iam-chamber-user.git?ref=master"
+  namespace   = "cp"
+  stage       = "staging"
+  name        = "chamber"
+  kms_key_arn = "arn:aws:kms:region:account-id:key/CMK"
```

Comment:
Add a realistic arn. It's hard to know what is a placeholder vs what is replaced.